packages/frontend-2/lib/workspaces/graphql/queries.ts

```diff
@@ -0,0 +1,31 @@
+import { graphql } from '~~/lib/common/generated/gql'
+
+export const workspacePageQuery = graphql(`
+  query WorkspacePageQuery(
+    $workspaceId: String!
+    $filter: WorkspaceProjectsFilter
+    $cursor: String
+  ) {
+    workspace(id: $workspaceId) {
+      name
```

Comment:
All of these fields are disconnected from the fragments and thus the Vue components that actually need them. No way to tell if any of this is necessary or not and gonna be a PITA to refactor when moving around/updating

https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#d40e40efc32f4e5ea67b68e848aadcd0